c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -23,43 +23,33 @@ Version 1.6
     cp /tmp/{{project}}/deploy/hooks/post-restore-code deploy/hooks
     chmod +x deploy/hooks/post-restore-code
 
-2. Removes the deprecated files:
-
-    git rm -r {{package}}/CONST_migration
-    git rm -r {{package}}/static/build
-    git rm config.yaml
-    git rm bootstrep.py
-    git rm -r *buildout*
-    git rm development.ini.in production.ini.in
-    git rm versions.cfg
-
-3. Starting from version 1.6 we use a new build system based on Make that will change
+2. Starting from version 1.6 we use a new build system based on Make that will change
    many of your habits, information are given at:
    http://docs.camptocamp.net/c2cgeoportal/1.6/integrator/make.html
 
    Main things to do:
    - Move your old `config.yaml` and your `vars` section of the old
-     `buildout.cfg` file to the new `{{package}}.yaml` file.
+     `buildout.cfg` file to the new `vars_{{package}}.yaml` file.
    - Move your special task from your old `buildout.cfg` file to your new
      `{{package}}.mk` file
    - The `README.rst`, `.gitignore` files will be overwritten.
      Please make sure to back your changes up before proceeding.
    - The `development.ini.in`, `production.ini.in` files will be replaced by
      the `development.ini.mako`, `production.ini.mako` files. If you lost
-     something it should probably be moved in the `{{package}}.yaml` file.
+     something it should probably be moved in the `vars_{{package}}.yaml` file.
    - The `print/template/print.mako.in` file should be moved to
      `print.yaml.mako`.
    - The build environments (`<user>.mk` files) must be recreated again.
      For multi project mode have a look on the documentation:
      http://docs.camptocamp.net/c2cgeoportal/master/integrator/create_application.html
 
-4. In the `setup.cfg` file you should remove all the `[compile_catalog]`,
+3. In the `setup.cfg` file you should remove all the `[compile_catalog]`,
    `[extract_messages]`, `[init_catalog]`, `[update_catalog]` sections.
 
-5. In the `setup.py` file you should remove the `message_extractors`
+4. In the `setup.py` file you should remove the `message_extractors`
    argument.
 
-6. In the {{package}}/__init__.py file do the following changes:
+5. In the {{package}}/__init__.py file do the following changes:
 
     -from c2cgeoportal import locale_negotiator
     +from c2cgeoportal import locale_negotiator, add_interface, INTERFACE_TYPE_SENCHA_TOUCH
```

Comment:
Actually, this part is not clear at all since it also adds interfaces for editing and routing (not only mobile).
Perhaps you could also precise where to paste those lines.